docs/contribute/code.md

```diff
@@ -143,7 +143,7 @@ npx sb@next link --local /path/to/local-repro-directory
 
 <details>
 
-<summary>`yarn dev` watches everything but is resource-intensive</summary>
```

Comment:
I removed the `dev` command entirely, let's all use `yarn build <packagenames> [--watch]` instead.